packages/node-sdk/src/flusher.ts

```diff
@@ -0,0 +1,64 @@
+import { constants } from "os";
+
+import { END_FLUSH_TIMEOUT_MS } from "./config";
+
+type Callback = () => Promise<void>;
+
+const killSignals = ["SIGINT", "SIGTERM", "SIGHUP", "SIGBREAK"] as const;
+
+export function subscribe(
+  callback: Callback,
+  timeout: number = END_FLUSH_TIMEOUT_MS,
+) {
+  let state: boolean | undefined;
+
+  const wrappedCallback = async () => {
+    if (state !== undefined) {
+      return;
+    }
+
+    state = false;
+
+    try {
+      const result = await Promise.race([
+        new Promise((resolve) => setTimeout(() => resolve(true), timeout)),
+        callback(),
+      ]);
```

Comment:
try running this program:
```
import client from "./bucket";

async function start() {
  await client.initialize();

  client.getFeature({}, "create-todos");

  console.log("done");
}

start();
```

you'll see that these lines add an additional 5s before existing

Reply:
check out `setTimeout().unref()`